api/db/populate-friendships.js

```diff
@@ -0,0 +1,85 @@
+const { getRandomSubset, getRandomDate, createPrompt } = require('./utils');
+
+const User = require('../models/User');
+const Friendship = require('../models/Friendship');
+
+let displayLogs;
+
+const createFriendship = async (friendshipParams) => {
+  const friendship = new Friendship({
+    requestant: friendshipParams.requestant,
+    receivant: friendshipParams.receivant,
+    accepted: friendshipParams.accepted,
+    acceptedAt: friendshipParams.acceptedAt,
+    lastMessageAt: friendshipParams.lastMessageAt,
+  });
+  await friendship.save();
+
+  if (displayLogs) {
+    console.log('\n', '\x1b[0m', `New friendship created: ${friendship}`);
+  }
+  return friendship;
+};
+
+const createFriendships = async (verbose) => {
+  console.log('\n', '\x1b[0m', 'Populating friendships collection...');
+  displayLogs = verbose;
+
+  let usersList = await User.distinct('_id');
+  const friendshipsArray = await Friendship.find();
+  let recordCount = friendshipsArray.length;
```

Comment:
This should be `const` instead of `let`. Also, maybe name it `documentsCount`?

Reply:
Also, as per one suggestion I've made below, this variable would only be used one time, so we can actually get rid of it and just evaluate it directly where it will be used.

Reply:
I agree that it can be changed to a `const` and name change, but this needs to be calculated before we start adding records to the array. It is used to calculate the number of records generated. Calculates `end array length - before array length`. See comment above about needing to use the `friendshipsArray` to avoid duplicate documents.